### PR TITLE
[CARBONDATA-1077] ColumnDict  and ALL_DICTIONARY_PATH must be used wi…

### PR DESCRIPTION
**Pull request by mohammadshahidkhan**
…th SINGLE_PASS='true'

### CODE REVIEW

File: integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala [hunk 1]
```diff
@@ -438,8 +438,9 @@ case class LoadTable(
         case "true" =>
           true
         case "false" =>
-          if (!StringUtils.isEmpty(allDictionaryPath)) {
-            true
+          if (StringUtils.isNotEmpty(allDictionaryPath) || StringUtils.isNotEmpty(columnDict)) {
```
Comment (kumarvishal09) [inline integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala]: Please add some comment

File: integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala [hunk 2]
```diff
@@ -448,8 +448,9 @@ case class LoadTable(
         case "true" =>
           true
         case "false" =>
-          if (!StringUtils.isEmpty(allDictionaryPath)) {
-            true
+          if (StringUtils.isNotEmpty(allDictionaryPath) || StringUtils.isNotEmpty(columnDict)) {
```
Comment (kumarvishal09) [inline integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala]: Please add some comment

File: integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala (hunk 1 above)
Comment (mohammadshahidkhan) [inline integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala]: done

**Comment by kumarvishal09** [comment]
except one small comment..LGTM

**Comment by mohammadshahidkhan** [comment]
retest this please

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2219/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/carbondata-pr-spark-1.6/91/

**Comment by mohammadshahidkhan** [comment]
all the test cases passed for both spark-2.1 and spark-1.6

**Comment by mohammadshahidkhan** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2313/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/carbondata-pr-spark-1.6/187/

**Comment by ravipesala** [comment]
LGTM

**Pull request closed without merging**